controllers/resource/resource_generator_test.go

```diff
@@ -303,3 +303,53 @@ func Test_createLivenessProbe(t *testing.T) {
 		require.Nil(t, actual)
 	})
 }
+
+func Test_getChownInitContainer(t *testing.T) {
+	t.Run("should return nil if volumes are only of type dogu-operator", func(t *testing.T) {
+		// given
+		dogu := &core.Dogu{Volumes: []core.Volume{{Clients: []core.VolumeClient{{Name: "k8s-dogu-operator"}}}}}
+
+		// when
+		container, err := getChownInitContainer(dogu, nil)
+
+		// then
+		require.Nil(t, container)
+		require.Nil(t, err)
```

Comment:
Is there a reason that sometimes `require.Nil(err)` is used and sometimes `require.Error(err)`. `require.NoError(err)` is also a thing.